dbschema/1.tables.sql

```diff
@@ -135,3 +135,26 @@ CREATE INDEX tbl_token_multi_ownerships_owner_address ON tbl_token_multi_ownersh
 CREATE INDEX tbl_token_multi_ownerships_latest_transfer_date ON tbl_token_multi_ownerships (latest_transfer_date);
 CREATE INDEX tbl_token_multi_ownerships_latest_transfer_value ON tbl_token_multi_ownerships (latest_transfer_value);
 CREATE INDEX tbl_token_multi_ownerships_latest_transfer_transaction_hash ON tbl_token_multi_ownerships (latest_transfer_transaction_hash);
+
+CREATE TABLE tbl_collection_hourly_activity (
+    id BIGSERIAL PRIMARY KEY,
+    created_date TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+    updated_date TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+    address TEXT NOT NULL,
+    date TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+    transfer_count NUMERIC(256, 0) NOT NULL,
```

Comment:
can we add one more column here pls - valued_transfer_count (or better name if you can think of one). the current transfer_count will count all transfers but it would be good to also have a number which tells us how many sales there were (i.e. transfers with a value)

Reply:
@Femi-Ogunkola not sure you saw this

Reply:
ahh okay cool